### PR TITLE
Updated the CSS on the login page

### PR DESCRIPTION
**Pull request by acron0**
Some of the CSS was a bit crusty so it's been updated to match the landing page

### CODE REVIEW

File: resources/public/css/login.css [hunk 1]
```diff
@@ -1 +1 @@
...
...at center center fixed;filter:progid:DXImageTransform.Microsoft.AlphaImageLoader(src='.myBackground.jpg',sizingMethod='scale');-ms-filter:progid:DXImageTransform.Microsoft.AlphaImageLoader(src='myBackground.jpg',sizingMethod='scale');background-size:cover;-moz-background-size:cover;-webkit-background-size:cover;-o-background-size:cover}.bg-attribution{color:#808080;font-size:.65em;font-family:monospace;float:right}.bg-attribution a{text-decoration:none;color:#fff}.title h1{font-size:5em;font-weight:700;padding:0 0 0 20px;margin:10px 0}.title h2{font-size:4em;font-weight:400;padding:0 20px 10px 20px;margin:0}#content-container{position:absolute;top:0;left:0;width:100%}#relative-container{position:relative;padding:5%;display:table-cell}#witan-login{margin-top:3em;margin-bottom:1em;padding:1px 30px 20px 20px;height:100%;width:300px}#witan-login #loading{color:#fff;margin:15px auto 0 auto;display:table}#witan-login input{margin-bottom:.5em;width:100%}#witan-login #forgotten-link{font-size:10px;vertical-align:text-top;text-align:right;float:right;color:#4caecf;cursor:pointer}#witan-login .forgotten-div #back-button{float:right}#witan-login #witan-copy{margin-top:20px;margin-bottom:20px}
```
Comment (otfrom) [inline resources/public/css/login.css]: Is this generated? If so I think we should remove it.

File: resources/public/css/style.css [hunk 2]
```diff
@@ -1 +0,0 @@
...Neue,Helvetica,Arial,sans-serif}body{font-size:16px;line-height:1.5}body,h1,h2,h3,h4,h5{font-family:Asap,Helvetica Neue,Helvetica,Arial,sans-serif}h1{color:#2d3e50}h2{color:#6b8aab}h3{color:#646464}hr{color:#abd3ff;background-color:#abd3ff}#witan-layout{padding-left:115px;left:0;position:relative}#witan-layout,#witan-menu,.witan-menu-link{transition:all .2s ease-out 0s}#witan-menu{box-shadow:0 1px 3px rgba(0,0,0,1);text-align:center;bottom:0;top:0;margin-left:-150px;background-color:#2d3e50;padding:.5em;position:fixed;left:150px}.witan-menu-item{color:#fff;padding-top:1em}.witan-menu-item a{text-decoration:none;color:#fff;border:medium none}.witan-menu-item a:hover{color:#6b8aab}#witan-page-title{margin:0;text-align:center;border-bottom:1px solid #aaa}#witan-page-title h1{font-size:4em;font-weight:400;margin:0}#witan-page-title h2{font-weight:300;color:#6b8aab;padding:0;margin-top:-20px;letter-spacing:3px}#witan-main-content{padding-left:30px;padding-right:30px}.witan-pattern-example{position:relative;min-height:100px}.witan-pattern-example pre{background-color:#f3f5c2;font-size:.8em}.witan-pattern-example-code{position:absolute;top:25%}.pure-menu-heading{color:#fff;font-size:20px}
```
Comment (otfrom) [inline resources/public/css/style.css:1]: Is this generated?

Comment (acron0) [inline resources/public/css/style.css:1]: Yes, using garden

**Comment by otfrom** [comment]
Remove generated files (or explain why they are in git) and :shipit:

**Pull request merged**